tools/prow-config-updater/config/config.go

```diff
@@ -48,6 +46,12 @@ const (
 	configPath         = "config"
 	configTemplatePath = "tools/config-generator/templates"
 
+	// Prow config folder names
+	prodConfigDirName             = "prod"
+	stagingConfigDirName          = "staging"
+	prowConfigDirName             = "prow"
+	prowBuildclusterConfigDirName = "build-cluster"
```

Comment:
It looks this const is not being used, do we need to keep it?

Reply:
yeah, it will be used in my next update.

Reply:
done